Specs/DateTime/Spanish/TimeExtractor.json

```diff
@@ -563,24 +563,24 @@
     ]
   },
   {
-    "Input": "Volvere a la tarde",
+    "Input": "Volvere a la media tarde",
```

Comment:
Would this be more correct as "a media tarde"? Not sure.

Reply:
replaced with "a media tarde" (it seems to be more common)